en/techdocs/api/client/hoodie.account.md

```diff
@@ -157,18 +156,25 @@ synchronized to.
 | HoodieAccountUnconfirmedError  | Account has not been confirmed yet |
 | HoodieAccountNotFoundError     | Account could not be found |
 | HoodieError                    | _A custom error can be set by plugins, e.g. the account could be blocked due to missing payments_ |
-| HoodieConnectionError | Could not connect to Server                 |
+| HoodieConnectionError | Could not connect to server                 |
+
+The **signIn** method tries to sign in the user to an existing account.
 
-<br />
+#### NOTES
 
+All Hoodie data that existed in the browser's local store before signin will be cleared on signin. To prevent data loss, you can pass **{moveData: true}** as the options argument, this will move current data (created anonymously or by another account) to the account the user signs in to.
```

Comment:
> or by another account

Did you test if that's the case? I'm not sure myself to be honest, if it's possible to sign in to a different account if you are currently signed in